Disable bogus file detection

- It doesn't work and there's no trivial way to fix
  it as the path is dependent on gettext version
  (e.g. ${LOCALBASE}/share/gettext-0.21/its/metainfo.its)
- It's not used anywhere

--- pentobi/unix/CMakeLists.txt.orig	2020-11-04 07:37:40 UTC
+++ pentobi/unix/CMakeLists.txt
@@ -6,16 +6,6 @@ if(NOT ITSTOOL)
 endif()
 get_filename_component(GETTEXT_BIN_DIR ${GETTEXT_MSGFMT_EXECUTABLE} DIRECTORY)
 get_filename_component(GETTEXT_INSTALL_DIR ${GETTEXT_BIN_DIR} DIRECTORY)
-find_file(METAINFO_ITS NAMES metainfo.its appdata.its
-    HINTS ${GETTEXT_INSTALL_DIR}/share/gettext/its
-    )
-if(NOT METAINFO_ITS)
-    message(FATAL_ERROR
-        "metainfo.its not found. Install appstream and/or use"
-        " -DMETAINFO_ITS=<file> to define the location of the metainfo.its"
-        " or appdata.its file."
-        )
-endif()
 find_program(RSVG_CONVERT rsvg-convert)
 if(NOT RSVG_CONVERT)
     message(FATAL_ERROR "rsvg-convert not found (install librsvg2-bin)")
